src/cloud-api-adaptor/podvm/files/etc/systemd/system/process-user-data.service

```diff
@@ -10,7 +10,8 @@ DefaultDependencies=no
 
 [Service]
 Type=oneshot
-ExecStart=/usr/local/bin/process-user-data provision-files
+EnvironmentFile=/etc/default/process-user-data
```

Comment:
That change looks like it belongs to the next commit (which is the one writing the file)